contracts/base/ModuleManager.sol

```diff
@@ -213,11 +213,31 @@ contract ModuleManager is Storage, Receiver, IModuleManagerEventsAndErrors {
     /// @param handler The address of the fallback handler to install.
     /// @param params The initialization parameters including the selector and call type.
     function _installFallbackHandler(address handler, bytes calldata params) internal virtual {
+        // Extracting the function selector from the parameters
         bytes4 selector = bytes4(params[0:4]);
+
+        // Extracting the call type from the parameters
         CallType calltype = CallType.wrap(bytes1(params[4]));
+
+        // Extracting the initialization data from the parameters
         bytes memory initData = params[5:];
-        if (_isFallbackHandlerInstalled(selector)) revert FallbackAlreadyInstalledForSelector(selector);
+
+        // Revert if the selector is either `onInstall(bytes)` (0x6d61fe70) or `onUninstall(bytes)` (0x8a91b0e3)
+        // These selectors are forbidden as they can lead to security vulnerabilities
+        // and unexpected behavior during fallback handler installation.
```

Comment:
can you describe what's the unexpected security behavior?

Reply:
"If it’s added a fallback method. Anyone can uninstall and reinstall the module. If it’s a validator, this will most likely pwn the account"